src/new_addon.ts

```diff
@@ -6,22 +6,23 @@ import { letUserPickItem } from './select_utils';
 import {
     cancel, readTextFile, writeTextFile, getWorkspaceFolders,
     addFolderToWorkspace, multiReplaceText, pathExists,
-    isValidPythonModuleName, renamePath
+    isValidPythonModuleName, renamePath, toTitleCase
 } from './utils';
 
-type AddonBuilder = (path: string, addonName: string, authorName: string) => Promise<string>;
+type AddonBuilder = (path: string, addonName: string, authorName: string, supportLegacy: boolean) => Promise<string>;
 
 const addonTemplateDir = path.join(templateFilesDir, 'addons');
+const manifestFile = path.join(addonTemplateDir, '..', 'blender_manifest.toml')
```

Comment:
Could also join `templateFilesDir` with `blender_manifest.toml`  instead of having to use `..`?